var/spack/repos/builtin/packages/neurodamus-model/package.py

```diff
@@ -152,7 +153,12 @@ def setup_run_environment(self, env):
             #  - NRNMECH_LIB_PATH the combined lib (used by neurodamus-py)
             #  - BGLIBPY_MOD_LIBRARY_PATH is the pure mechanism
             #        (used by bglib-py)
-            if '_nd.' in libnrnmech_name:
+            if 'darwin' in self.spec.architecture:
+                dso_suffix = ".dylib"
+            else:
+                dso_suffix = ".so"
+
+            if 'libnrnmech' + dso_suffix in libnrnmech_name:
```

Comment:
why not simply `libnrnmech.` (with the dot?) sufixes were not separated by dot

Reply:
Good catch. It can simply be `libnrnmech.` 👍